File: tests/machine/x/fortran/append_builtin.f90
program append_builtin
  integer, dimension(:), allocatable :: a
  integer, dimension(:), allocatable :: b
  allocate(a(2))
  a = (/1,2/)
  allocate(b(3))
  b(1:2) = a
  b(3) = 3
  print *, b
end program append_builtin

File: tests/human/x/pas/append_builtin.pas
program AppendBuiltin;
var
  a: array of integer;
  i: integer;
begin
  SetLength(a, 2);
  a[0] := 1;
  a[1] := 2;
  SetLength(a, 3);
  a[2] := 3;
  for i := 0 to High(a) do
  begin
    if i > 0 then Write(' ');
    Write(a[i]);
  end;
  Writeln;
end.
